spec/models/region_spec.rb

```diff
@@ -61,24 +61,17 @@
       expect(district.block_regions.first.reportable_children).to match_array([facility.region])
     end
 
-    it "excludes states and blocks for other countries" do
-      allow(CountryConfig).to receive(:current).and_return(CountryConfig.for(:BD)).at_least(:once)
+    it "is everything for Bangladesh" do
+      expect(CountryConfig).to receive(:current).and_return(CountryConfig.for(:BD)).at_least(:once)
 
       org = Seed.seed_org.region
-      state = FactoryBot.create(:region, :state, reparent_to: org)
+      state = FactoryBot.create(:region, :state, reparent_to: Seed.seed_org.region)
```

Comment:
You can use the local org here, just a tiny bit more clear.

```suggestion
      state = FactoryBot.create(:region, :state, reparent_to: org)
```